disnake/ext/tasks/__init__.py

```diff
@@ -325,9 +324,7 @@ def start(self, *args: Any, **kwargs: Any) -> asyncio.Task[None]:
             args = (self._injected, *args)
 
         if self.loop is MISSING:
-            with warnings.catch_warnings():
-                warnings.simplefilter("ignore", DeprecationWarning)
-                self.loop = asyncio.get_event_loop()
+            self.loop = disnake.utils.get_event_loop()
```

Comment:
The docstring still mentions `asyncio.get_event_loop`, could we adjust this to clarify that it's not relying on deprecated behavior? (while still doing effectively the same thing that `asyncio.get_event_loop` did)

Reply:
https://github.com/DisnakeDev/disnake/pull/1337/commits/7de89eb04997441f581a996234d3873d97dd50b4